CHANGELOG.md

```diff
@@ -6,16 +6,36 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [1.5.0] - 2022-12-06
+
+### Changed
+- Use the /v1/batch endpoint for batching events (this should be backward compatible according to Segment) https://github.com/whatthewhat/simple_segment/pull/40 by [@sgallag-insta](https://github.com/sgallag-insta)
```

Comment:
please remove the comment for segment

Reply:
This is from the forked. Should we remove them?

Reply:
Let's remove the CHANGELOG. It doest match our releases.